specs/schema.TEStribute.openapi.yaml

```diff
@@ -0,0 +1,96 @@
+openapi: "3.0.0"
+info:
+  version: 0.0.1
+  title: TEStribute OpenAPI specification
+  contact:
+    name: 'Elixir Europe'
+    email: 'alexander.kanitz@alumni.ethz.ch'
+  license:
+    name: Apache 2.0
+    url: 'https://github.com/elixir-europe/TEStribute/blob/dev/LICENSE'
+paths:
+  /rank_services:
+    post:
+      operationId: rank_services
+      description: Creates a new pet in the store.  Duplicates are allowed
+      requestBody:
+        description: Pet to add to the store
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/rank_services_requirements'
```

Comment:
Could you please put the names in CamelCase or camelCase as it seems to be a convention for models, or not? Does OpenAPI 3.0 guide says anything about it? If you replace, please do consistently for all models